net-misc/frr/frr-9.0.1.ebuild

```diff
@@ -14,7 +14,7 @@ S="${WORKDIR}/frr-${P}"
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="~amd64 ~arm64 ~x86"
+KEYWORDS="amd64 ~arm64 ~x86"
```

Comment:
We don't let users stabilize packages, but it should go through arch testers. Can you file a stabilization bug in bugzilla? Let me know if you're not familiar with that.

Oh and if I read the CVE correctly, 9.0.1 is still affected. So we should insta-stabilize 9.0.2 instead?

Reply:
We should, I just didn’t want to stabilise 9.0.2 right away :)